components/docusign/docusign.app.js

```diff
@@ -0,0 +1,129 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "docusign",
+  propDefinitions: {
+    account: {
+      type: "string",
+      label: "Account",
+      description: "",
```

Comment:
Does Zapier have any description of the account? If this prop is obvious to the user, feel free to also just remove the description altogether (vs. keeping the empty string)

Reply:
Updated